cumulusci/cumulusci.yml

```diff
@@ -85,6 +85,20 @@ tasks:
             path: src
             revert_path: src.orig
         group: Salesforce Metadata
+    create_permission_set:
+        group: Metadata Transformations
+        description: Creates a Permission Set with specified User Permissions and assigns it to the running user.
+        class_path: cumulusci.tasks.salesforce.create_permission_sets.CreatePermissionSet
+    create_bulk_data_permission_set:
+        group: Data Operations
+        description: "Creates a Permission Set with the Hard Delete and Set Audit Fields user permissions. NOTE: the org setting to allow Set Audit Fields must be turned on."
```

Comment:
Is there anything we can link to with an example of that setting?

Reply:
Yes. I'll add a short section to the Bulk Data docs to call out that this task can help, and link to the Salesforce help article.

Reply:
Documentation added with link.